google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -166,12 +163,6 @@
   private EnhancedBigtableStubSettings(Builder builder) {
     super(builder);
 
-    if (DIRECT_PATH_ENDPOINT.equals(builder.getEndpoint())) {
-      logger.warning(
-          "Connecting to Bigtable using DirectPath."
```

Comment:
is there anywhere we can keep this logging information to confirm (in tests, for example) that we are connecting using DirectPath?

Reply:
For tests we inject hooks into netty to ensure that directpath is used, also the integration tests generate verbose grpc logs which would show directpath usage. For prod usage, we dont really have a good way to introspect it as its a grpc level feature